templates/end_user_devices/smartphone.yaml

```diff
@@ -4,5 +4,6 @@ template:
   type: end_user_device
   draw_watts: 0.77
   production_emissions_gco2e_per_duration_s: 0.0052
+  streaming_resolution: medium
```

Comment:
should the streaming_resolution be included here by connection type rather than 1 value for the end user device independent of connection type?

Reply:
Right now, I've kept things simple since it is dependent purely on device. I did note that would likely change. I'll push a first cut of this and then can always make more complex

Reply:
Update